chapter_01/hinglish/introduction.ipynb

```diff
@@ -0,0 +1,1346 @@
+{
```

Comment:
<p><s>depend(</s>nirbhar<s>)</s></p><p> </p><p>common suggestion - these type of translation is not required for common words. either of english or hindi can be kept.. there are many such cases, i've not mentioned everywhere.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/abhiabhi94/learn-python/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='35'/>

Reply:
<p>okay. i just didn't want to assume that the reader is too much versed with a lot of english words. Since we have been using english for some time now, it seems it could be easier for us to assume some words could be easily understood by the audience while this may be not true.</p><p> </p><p>I think we can maybe keep this in the first edition, and remove them if necessary in future versions. </p><p> </p><p>What do you suggest?</p><p> </p>

Reply:
<p>I can feel your good intention, but this breaks the rhythm for the readers who already know the meaning while reading.  In my opinion, in the text we can make these words bold, and it can be added in the glossary section in the end. Also, I'd like to someone else opinion on this.</p>